.github/workflows/codeql-analysis-linux.yml

```diff
@@ -0,0 +1,48 @@
+name: "CodeQL for Linux"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '23 11 * * 6'
+
+jobs:
+  analyze-linux:
+    name: Analyze Linux (Ubuntu 18.04)
+    runs-on: ubuntu-latest
```

Comment:
You could use a [matrix](https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix) to run the steps on every OS here.

Reply:
Is done :)